app/auth/FogotPassword/page.jsx

```diff
@@ -0,0 +1,90 @@
+"use client";
+import { Box, Button, TextField } from "@mui/material";
+import Layout from "../layput";
+import imgageicon from '../images/forgotIcon.png'
+import styles from './page.module.css'
+import { useState } from "react";
+import Link from "next/link";
+
+export default () => {
+
+    const [emails, setEmail] = useState('')
+    const [emilValid, setEmailValid] = useState(true)
+
+
+    // regex check valid email
+    const validateEmail = (email) => {
```

Comment:
You could move this into a new folder utils/utils.js.
As we could this function many times